### PR TITLE
Event balancing

### PR DESCRIPTION
**Pull request by ThatDan123**
-Fixes up for event chances.

-Reduces immovable rod and meteors to 0.5%, and ups time between events to 20 mins.

-Adds new melee event and cookie event.

### CODE REVIEW

File: UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			{
+				if (player.Script.IsDeadOrGhost) continue;
+
+				var slot = player.Script.Equipment.ItemStorage.GetActiveHandSlot();
+
+				if (slot == null) continue;
+
+				if (slot.Item == null)
+				{
+					var item = Spawn.ServerPrefab(itemList[UnityEngine.Random.Range(1, itemList.Count)], player.Script.WorldPos, player.Script.gameObject.transform.parent, player.Script.transform.rotation);
```
Comment (vlandemart) [inline UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs]: Why does Random.Range start from 1?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs]: oh god, good point.

**Review by vlandemart — APPROVED** [review]

File: UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+				if (slot.Item == null)
+				{
+					var item = Spawn.ServerPrefab(itemList[UnityEngine.Random.Range(0, itemList.Count)], player.Script.WorldPos, player.Script.gameObject.transform.parent, player.Script.transform.rotation);
+
+					Inventory.ServerAdd(item.GameObject.GetComponent<Pickupable>(), slot);
+				}
+				else
+				{
+					Spawn.ServerPrefab(itemList[UnityEngine.Random.Range(1, itemList.Count)], player.Script.WorldPos, player.Script.gameObject.transform.parent, player.Script.transform.rotation);
```
Comment (CodeKalias) [inline UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs:44]: Another Random.Range starting at 1 here

Comment (vlandemart) [reply UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs:44]: my bad, missed that one

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventGiveItems.cs:44]: dont worry ive got another PR coming

**Pull request merged**